package.json

```diff
@@ -9,7 +9,7 @@
     "dev": "next dev -H 0.0.0.0"
   },
   "dependencies": {
-    "chart.js": "^2.9.3",
+    "chartjs-chart-box-and-violin-plot": "^2.4.0",
```

Comment:
Im ignoring all changes to package.json as we're not using npm but yarn install/add